interview-questions.md

```diff
@@ -10,40 +10,40 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 Similarities
 
-1.
-2.
-3.
+1. They both do similiar logic within the code. meaning they both support conditional statements, loops, functions/methods, and others.
+2. both are interpreted languages so code is executed line by line
+3. both are object oriented languages. meaning both support creating and manipulating objects with properties and methods
```

Comment:
Ruby is definitely object-oriented. JavaScript uses prototypical inheritance so isn't considered a classic oop language.